src/Xamarin.Android.Build.Tasks/Tasks/Aapt2Compile.cs

```diff
@@ -111,7 +122,7 @@ protected string[] GenerateCommandLineCommands (string fileOrDirectory, bool isD
 				cmd.Add (GetFullPath (ResourceSymbolsTextFile));
 			}
 			if (isDirectory) {
-				cmd.Add ("--dir");
+				cmd.Add (isArchive ? "--zip" : "--dir");
 				cmd.Add (GetFullPath (fileOrDirectory).TrimEnd ('\\'));
```

Comment:
Considering that this PR is in the context of "Chinese character set support", and that this is a command-line argument to `aapt2`, should this *really* use `GetFullPath(fileOrDirectory)` at all anymore?

I'm kinda surprised that this works at all, in that the PRs are green…